### PR TITLE
PWN-6554 - provide showToast everywhere!

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6554

## Description of Work

Provide showToast in MvpView.kt and use it to show max on new Send

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt [hunk 1]
```diff
@@ -7,6 +7,10 @@ import com.google.android.material.snackbar.Snackbar
 interface BaseFragmentContract {
     fun overrideEnterAnimation(@AnimRes animation: Int)
     fun overrideExitAnimation(@AnimRes animation: Int)
+    fun showToast(
+        message: String? = null,
+        @StringRes messageResId: Int? = null
+    )
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt:13]: use `TextContainer` instead of two nullable types

**Review by 1sakov — CHANGES_REQUESTED** [review]

**Comment by gslevinkov** [comment]
can i see the design of such toast?

File: app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt:13]: Waaaat ?

**Comment by kabramovp2p** [comment]
> can i see the design of such toast?

take this
image

File: app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt (hunk 1 above)
Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt:13]: You'll be able to pass 
`TextContainer(R.string.some_string)` or 
`TextContainer("Some string")`

And get the value by `getString(context)`

Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt:13]: ohh i see
Djezd'

Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/common/mvp/BaseFragmentContract.kt]: @1sakov fixed

**Review by 1sakov — APPROVED** [review]

**Pull request merged**